static/src/javascripts/lib/add-event-listener.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+let supportsOptions = false;
+
+try {
+    const opts = Object.defineProperty(
+        {},
+        'passive',
+        ({
+            get() {
+                supportsOptions = true;
+            },
+        }: Object) // https://github.com/facebook/flow/issues/285
+    );
+    window.addEventListener('test', null, opts);
+} catch (e) {
+    /* noop */
+}
+
+const addEventListener = (
+    node: Node,
+    name: string,
+    handler: Function,
+    options: Object = {}
+): void | false => {
```

Comment:
yeah good catch, i had it returning `false` if node was undefined before, left it accidentally